domains/groteam.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Groteam1001",
+        "email": "groteam2@outlook.com"
+    },
+    "records": {
+        "A": "45.131.65.38"
```

A records is an Array
```suggestion
        "A": ["45.131.65.38"]
```